fud2/src/lib.rs

```diff
@@ -760,11 +762,16 @@ pub fn build_driver(bld: &mut DriverBuilder) {
     let wrapper_setup = bld.setup("YXI and AXI generation", |e| {
         // Define a `gen-axi` rule that invokes our Python code generator program.
         // For now point to standalone axi-generator.py. Can maybe turn this into a rsrc file?
-        e.config_var_or(
-            "axi-generator",
-            "axi.generator",
-            "$calyx-base/yxi/axi-calyx/axi-generator.py",
-        )?;
+        let dynamic =
+            e.config_constrained_or("dynamic", vec!["true", "false"], "false")?;
+        let generator_path = if FromStr::from_str(&dynamic)
+            .expect("The dynamic flag should be either 'true' or 'false'.")
```

Comment:
I know you just added the "enum-style" string configuration in #2135, but it occurs to me that we could also make this a proper Boolean. This would be easy in the TOML (`dynamic = true` or whatever will just do it), but it would require more work to make the CLI work the same way. So no need to change for now IMO!